src/main/java/com/google/sps/servlets/GetImageServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.googl.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * When the user submits the form, Blobstore processes the file upload and then forwards the request
+ * to this servlet. This servlet then returns the image fetched from Blobstore.
+ */
+@WebServlet("/get-image")
+public class GetImageServlet extends HttpServlet {
+  private static final String BLOBKEY_PARAMETER = "blobkey";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
```

Comment:
So I have seen the `DatastoreService` be stored as a [member variable](https://github.com/googleinterns/step144-2020/blob/6856bbf6d591639d073242e9f6ae1cd64abdb9c7/src/main/java/com/google/sps/servlets/CareerQuizServlet.java#L25). Should we do the same with `BlobstoreService`, for consistency?